### PR TITLE
Tournament creation in menu bar

### PR DESCRIPTION
**Pull request by Theepanguna**
I have created a tournament create and tournament display menu bar. This is v1, let me know the feedback so I will work furthermore

### CODE REVIEW

**Comment by Theepanguna** [comment]
I have created a tournament create and tournament display menu bar. This is v1, let me know the feedback so I will work furthermore

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -3,13 +3,15 @@ import { RouterModule, Routes } from '@angular/router';
...
 import { RegisterComponent } from './views/register/register.component';
+import { TournamentCreateComponent } from './views/tournaments/tournament-create/tournament-create.component';
 import { TournamentsComponent } from './views/tournaments/tournaments.component';
 
 const routes: Routes = [
   { path: '', component: HomeComponent },
   { path: 'tournaments', component: TournamentsComponent },
   { path: 'register', component: RegisterComponent },
   { path: 'contact', component: ContactComponent },
+  { path: 'tournamentCreate', component: TournamentCreateComponent },
```
Comment (SebastianSilvaDev) [inline src/app/app-routing.module.ts]: Ok this is wrong, you can use child routes instead

File: src/app/views/tournaments/tournament-create/tournament-create.component.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    </mat-card-header>
+    <mat-card-content>
+      <form style="display: flex; flex-direction: column;">
+        <mat-form-field appearance="fill">
+          <mat-label >Tournament Name</mat-label>
+          <input matInput type="text" name="name">
+        </mat-form-field>
+        
+            <mat-label>Number of Players</mat-label>
+            <mat-radio-group ([ngModel])="selectedPlayer">
```
Comment (SebastianSilvaDev) [inline src/app/views/tournaments/tournament-create/tournament-create.component.html]: Dont use ngModel yet, thats for FormGroup and Validations

**Review by SebastianSilvaDev — CHANGES_REQUESTED** [review]
Overall good, some changes will be needed

**Pull request merged**